services/src/main/java/org/keycloak/services/managers/RealmManager.java

```diff
@@ -424,7 +426,8 @@ private void setupAccountManagement(RealmModel realm) {
             accountClient.setFullScopeAllowed(false);
 
             accountClient.setRootUrl(Constants.AUTH_BASE_URL_PROP);
-            String baseUrl = "/realms/" + realm.getName() + "/account/";
+
+            String baseUrl = UriBuilder.fromPath("/realms/{realmName}/account/").build(realm.getName()).toString();
```

Comment:
Same here, seems something we repeat often enough for the path to be stored as a constant somewhere.

Reply:
Same, I did string concatenation but with the realm name encoded.